code/formatters.py

```diff
@@ -124,8 +124,8 @@ def every_word(word_func):
 
 # This is the mapping from spoken phrases to formatters
 formatters_words = {
-    "allcaps": formatters_dict["ALL_CAPS"],
-    "alldown": formatters_dict["ALL_LOWERCASE"],
+    "all caps": formatters_dict["ALL_CAPS"],
+    "all down": formatters_dict["ALL_LOWERCASE"],
     "camel": formatters_dict["PRIVATE_CAMEL_CASE"],
     "dotted": formatters_dict["DOT_SEPARATED"],
     "dubstring": formatters_dict["DOUBLE_QUOTED_STRING"],
```

Comment:
Should we add a space here as well? I have had slight troubles with dubstring in the past.
```suggestion
    "dub string": formatters_dict["DOUBLE_QUOTED_STRING"],
```